scripts/gcs/a6000_camera_management/get_images.py

```diff
@@ -1,26 +0,0 @@
-from pysony import SonyAPI, payload_header
-import urllib2
-import thread
-import time
-import shutil
-import os
-
-class ClassName(object):
```

Comment:
Python class names should be descriptive.

Reply:
This file has been removed.